### DISCUSSION TITLE
librechat:0.6.10 image unavailable for arm64

### DISCUSSION THREAD

**Discussion by Nindaleth** [danny-avila/LibreChat#1831] [Troubleshooting]
### What happened?

Hello, I'm a happy user of LibreChat 0.6.9 on an RPi4. I see a new release has been done since but I'm not able to pull the image. It seems the latest release doesn't provide an `arm64` image.

### Steps to Reproduce

* `docker image pull ghcr.io/danny-avila/librechat:latest`
* compare [v0.6.9](https://github.com/danny-avila/LibreChat/pkgs/container/librechat/178409192?tag=v0.6.9) and [v0.6.10](https://github.com/danny-avila/LibreChat/pkgs/container/librechat/180057928?tag=v0.6.10) arm64 availability in the Arch tab

### Relevant log output

```shell
$ docker image pull ghcr.io/danny-avila/librechat:latest
latest: Pulling from danny-avila/librechat
no matching manifest for linux/arm64/v8 in the manifest list entries
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I could not get the docker workflow to work for an arm64 build without removing the addition of curl in the main Dockerfile. I may fix this later on.

For now, Use librechat-api:latest for an arm64 image. You will have to use the `deploy-compose` file or a variation of it.

https://github.com/danny-avila/LibreChat/pkgs/container/librechat-api
